### PR TITLE
Logout-Homepage View

### PR DESCRIPTION
**Pull request by ohylime**
Home page header and images rendered in component. So far this is what it looks like. Will be creating new branch after to work on footer so it doesn't cover the other components. 
screenshot

### CODE REVIEW

File: src/components/home/logged-out.jsx [hunk 1]
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
       <div>
         {this.renderTitleBar()}
         <div className="body-with-title-bar container">
-          logged out wew
+        <div className="description">
+          <div className="header1">
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: make header1 and header2 a single h1 tag

File: src/components/home/logged-out.jsx [hunk 2]
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
...
+          the alumni-powered
+          </div>
+          <div className="header2">scholarship</div>
+          <div> built by </div>
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: put this in a singular div, with two divs to break up two lines

File: src/components/home/logged-out.jsx [hunk 3]
```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
...
+          <div> fullstack academy alumni for future alumni
+          </div>
+        </div>
+          <div className="row">
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: for readability can you break this out to it's own submethod? Something like `renderImages`

Same with the header portion

File: src/stylesheets/components/logged-out.less [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.description {
+  text-align: center;
+  margin : 9rem
+}
+
+.header1 {
```
Comment (covlllp) [inline src/stylesheets/components/logged-out.less]: header1 and header2 don't need to be idfferent classes right?

**Review by covlllp — CHANGES_REQUESTED** [review]

File: src/components/home/logged-out.jsx (hunk 1 above)
Comment (ohylime) [reply src/components/home/logged-out.jsx]: changed~

File: src/components/home/logged-out.jsx (hunk 2 above)
Comment (ohylime) [reply src/components/home/logged-out.jsx]: okay

File: src/stylesheets/components/logged-out.less (hunk 4 above)
Comment (ohylime) [reply src/stylesheets/components/logged-out.less]: combined them

File: src/components/home/logged-out.jsx (hunk 3 above)
Comment (ohylime) [reply src/components/home/logged-out.jsx]: submethods made!

File: src/components/home/logged-out.jsx [hunk 5]
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
     return <TitleBar rightItems={rightItems} />;
   }
 
+  renderLogoutViewHeader() {
+    const header = (
+      <div className="description">
+        <h1 className="header">
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: make this just one h1

File: src/components/home/logged-out.jsx [hunk 6]
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
...
+          fullstack academy alumni for future alumni
+        </p>
+      </div>);
+    return header;
+  }
+
+  renderLogoutViewImages() {
+    const images = (
+      <div className="row">
+        <div className="col-sm-6 col-md-3 col-lg-3">
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: make this a function that you just call 3x

File: src/components/home/logged-out.jsx [hunk 7]
```diff
@@ -25,12 +24,50 @@ export default class LoggedOutHome extends React.PureComponent {
...
+    return images;
+  }
+
   render() {
     return (
       <div>
         {this.renderTitleBar()}
         <div className="body-with-title-bar container">
-          logged out wew
+          {this.renderLogoutViewHeader()}
```
Comment (covlllp) [inline src/components/home/logged-out.jsx]: nit: don't include `Logout`, just have `renderHeader` and `renderImages`

File: src/stylesheets/components/logged-out.less [hunk 8]
```diff
@@ -0,0 +1,14 @@
+.description {
+  text-align: center;
```
Comment (covlllp) [inline src/stylesheets/components/logged-out.less]: use the `.text-align` class that's made in `main.less`

**Review by covlllp — CHANGES_REQUESTED** [review]
component cleanup

**Review by covlllp — APPROVED** [review]
lgtm

**Pull request merged**